<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-UJIBV1G" failures="8" tests="25" name="ULP BFF" time="246.181" errors="0" timestamp="2023-06-12T16:13:59 IST">
  <testcase classname="com.api.script.DigilockerEwalletTest" name="API_01_VerifyUserIsAbleToGenerateDigiUrl" time="3.696"/>
  <testcase classname="com.api.script.DigilockerEwalletTest" name="API_02_VerifyUserIsAbleToGenerateAuthCode" time="69.995"/>
  <testcase classname="com.api.script.DigilockerEwalletTest" name="API_03_VerifyUserIsAbleToGenerateToken" time="1.609"/>
  <testcase classname="com.api.script.DigilockerEwalletTest" name="API_04_VerifyUserIsAbleToUenerateUuid" time="0.783"/>
  <testcase classname="com.api.script.DigilockerEwalletTest" name="API_05_VerifyUserIsAbleToGetStateList" time="1.85"/>
  <testcase classname="com.api.script.DigilockerEwalletTest" name="API_06_VerifyUserIsAbleToGetDistrictList" time="1.039"/>
  <testcase classname="com.api.script.DigilockerEwalletTest" name="API_07_VerifyUserIsAbleToGetBlockList" time="2.249">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index 8 out of bounds for length 6">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index 8 out of bounds for length 6
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
at java.base/java.util.Objects.checkIndex(Objects.java:359)
at java.base/java.util.ArrayList.get(ArrayList.java:427)
at com.api.view.APIview.getBlockCode(APIview.java:538)
at com.api.script.DigilockerEwalletTest.API_07_VerifyUserIsAbleToGetBlockList(DigilockerEwalletTest.java:168)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- API_07_VerifyUserIsAbleToGetBlockList -->
  <testcase classname="com.api.script.DigilockerEwalletTest" name="API_08_VerifyUserIsAbleToGetSchoolList" time="53.722"/>
  <testcase classname="com.api.script.DigilockerEwalletTest" name="API_09_VerifyUserIsAbleToRegisterStudent" time="5.067">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.generic.WrapperFunctions.logReporter(WrapperFunctions.java:276)
at com.api.view.commonMethodsView.validateStatusCode(commonMethodsView.java:25)
at com.api.script.DigilockerEwalletTest.API_09_VerifyUserIsAbleToRegisterStudent(DigilockerEwalletTest.java:220)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- API_09_VerifyUserIsAbleToRegisterStudent -->
  <testcase classname="com.api.script.DigilockerEwalletTest" name="API_10_VerifyUserIsAbleToGetStudentData" time="0.462">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.generic.WrapperFunctions.logReporter(WrapperFunctions.java:301)
at com.api.view.APIview.getSuccessMessageStudentData(APIview.java:633)
at com.api.script.DigilockerEwalletTest.API_10_VerifyUserIsAbleToGetStudentData(DigilockerEwalletTest.java:233)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- API_10_VerifyUserIsAbleToGetStudentData -->
  <testcase classname="com.api.script.DigilockerEwalletTest" name="API_11_VerifyThatUserIsAbleToGenerateAuthCodeAndTokenForUsername" time="28.543"/>
  <testcase classname="com.api.script.DigilockerEwalletTest" name="API_12_verifyUserIsAbleToLogoutFromDigilocker" time="0.58"/>
  <testcase classname="com.api.script.DigilockerEwalletTest" name="API_13_SBRCSearchStudent" time="0.416">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;java.util.ArrayList.get(int)&amp;quot; because the return value of &amp;quot;com.api.pojoResponse.SBRCSearchStudentResponse.getResult()&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "java.util.ArrayList.get(int)" because the return value of "com.api.pojoResponse.SBRCSearchStudentResponse.getResult()" is null
at com.api.view.APIview.getStudentOsid(APIview.java:626)
at com.api.script.DigilockerEwalletTest.API_13_SBRCSearchStudent(DigilockerEwalletTest.java:280)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- API_13_SBRCSearchStudent -->
  <testcase classname="com.api.script.DigilockerportalTest" name="API_01_VerifyUserIsAbleToGenerateDigiUrlForPortal" time="0.435"/>
  <testcase classname="com.api.script.DigilockerportalTest" name="API_02_VerifyUserIsAbleToGenerateAuthCode" time="26.838"/>
  <testcase classname="com.api.script.DigilockerportalTest" name="API_03_VerifyUserIsAbleToGenerateToken" time="2.697"/>
  <testcase classname="com.api.script.DigilockerportalTest" name="API_04_VerifyUserIsAbleToGenerateUuidPortal" time="0.729"/>
  <testcase classname="com.api.script.DigilockerportalTest" name="API_09_VerifyUserIsAbleToVerifyUdiseDetails" time="1.365"/>
  <testcase classname="com.api.script.DigilockerportalTest" name="API_10_VerifyUserIsAbleToRegisterTeacher" time="5.907">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.generic.WrapperFunctions.logReporter(WrapperFunctions.java:276)
at com.api.view.commonMethodsView.validateStatusCode(commonMethodsView.java:25)
at com.api.script.DigilockerportalTest.API_10_VerifyUserIsAbleToRegisterTeacher(DigilockerportalTest.java:159)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- API_10_VerifyUserIsAbleToRegisterTeacher -->
  <testcase classname="com.api.script.DigilockerportalTest" name="API_11_VerifyUserIsAbleToGetSchoolDetails" time="0.45">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;com.api.pojoResponse.ResultSchoolDetails.getOsid()&amp;quot; because the return value of &amp;quot;com.api.pojoResponse.SchoolDetailsResponse.getResult()&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.api.pojoResponse.ResultSchoolDetails.getOsid()" because the return value of "com.api.pojoResponse.SchoolDetailsResponse.getResult()" is null
at com.api.view.APIview.getOsidSchool(APIview.java:778)
at com.api.script.DigilockerportalTest.API_11_VerifyUserIsAbleToGetSchoolDetails(DigilockerportalTest.java:177)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- API_11_VerifyUserIsAbleToGetSchoolDetails -->
  <testcase classname="com.api.script.DigilockerportalTest" name="API_12_verifyUserIsAbleToLogoutFromDigilocker" time="0.66"/>
  <testcase classname="com.api.script.DigilockerportalTest" name="API_13_VerifyUserIsAbleToGetTeacherData" time="0.468">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.generic.WrapperFunctions.logReporter(WrapperFunctions.java:301)
at com.api.view.APIview.getSuccessMessageStudentData(APIview.java:633)
at com.api.script.DigilockerportalTest.API_13_VerifyUserIsAbleToGetTeacherData(DigilockerportalTest.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- API_13_VerifyUserIsAbleToGetTeacherData -->
  <testcase classname="com.api.script.DigilockerportalTest" name="API_14_VerifyThatUserIsAbleToGenerateAuthCodeAndTokenForUsername" time="35.997"/>
  <testcase classname="com.api.script.DigilockerportalTest" name="API_15_VerifyUserIsAbleToGetDashboardCount" time="0.405">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.generic.WrapperFunctions.logReporter(WrapperFunctions.java:276)
at com.api.view.commonMethodsView.validateStatusCode(commonMethodsView.java:25)
at com.api.script.DigilockerportalTest.API_15_VerifyUserIsAbleToGetDashboardCount(DigilockerportalTest.java:243)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- API_15_VerifyUserIsAbleToGetDashboardCount -->
  <testcase classname="com.api.script.DigilockerportalTest" name="API_16_VerifyUserIsAbleToFindStudentForApproval" time="0.007"/>
</testsuite> <!-- ULP BFF -->
